package.json

```diff
@@ -1140,7 +1140,7 @@
     "fp-ts": "^2.3.1",
     "get-port": "^5.0.0",
     "getopts": "^2.2.5",
-    "getos": "^3.1.0",
+    "getos": "^3.2.1",
```

Comment:
`3.2.1` was published 5 years ago and doesn't look like it's actively maintained, going against the recommended 3rd party dependency guidelines.
Can we not switch entirely to native node `os`?